src/diffusers/loaders.py

```diff
@@ -1073,6 +1136,17 @@ def load_lora_into_text_encoder(cls, state_dict, network_alpha, text_encoder, lo
                             f"{name}.out_proj.lora_linear_layer.down.weight"
                         ] = text_encoder_lora_state_dict.pop(f"{name}.to_out_lora.down.weight")
 
+                if text_encoder_lora_state_dict:
+                    for name, _ in text_encoder_aux_modules(text_encoder):
+                        for direction in ["up", "down"]:
+                            for layer in ["fc1", "fc2"]:
```

Comment:
There seems to be this new identifier called "hada" in the SDXL LoRAs (see https://github.com/huggingface/diffusers/issues/4133#issuecomment-1641209337). 

Do we want to account for that too? Okay to leave it for now as well.

Reply:
> There seems to be this new identifier called "hada" in the SDXL LoRAs (see [#4133 (comment)](https://github.com/huggingface/diffusers/issues/4133#issuecomment-1641209337)).
> 
> Do we want to account for that too? Okay to leave it for now as well.

hada and lokr are from here
https://github.com/KohakuBlueleaf/LyCORIS

Reply:
Let's do them in subsequent PRs I think (just to keep the scope small and well tested, since this is mainly a revival of #3756)